dev/Materials/Acrylic/AcrylicBrush_rs2_themeresources.xaml

```diff
@@ -477,7 +488,17 @@
                 FallbackColor="#F3F3F3"
                 BackgroundSource="HostBackdrop"/>
             <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="#FFFFFF" Opacity="0.5"/>
-
+            <local:AcrylicBrush x:Key="AccentAcrylicBackgroundFillColorDefaultBrush"
+                TintColor="{ThemeResource SystemAccentColorLight3}"
+                TintOpacity="0.8"
+                FallbackColor="{ThemeResource SystemAccentColorLight3}"
+                BackgroundSource="HostBackdrop"/>
+            <local:AcrylicBrush x:Key="AccentAcrylicBackgroundFillColorBaseBrush"
+                TintColor="{ThemeResource SystemAccentColorLight2}"
+                TintOpacity="0.8"
+                FallbackColor="{ThemeResource SystemAccentColorLight2}"
+                BackgroundSource="HostBackdrop"/>
+            <SolidColorBrush x:Key="AccentLayerFillColorDefaultBrush" Opacity="0.25" Color="{ThemeResource SystemAccentColorLight3}"/>
```

Comment:
>AccentLayerFillColorDefaultBrush [](start = 36, length = 32)

Don't we need an entry for this brush in High Contrast ?

Reply:
Yes, we do good point. Added it now.